nilearn/_utils/niimg.py

```diff
@@ -0,0 +1,168 @@
+"""
+Neuroimaging file input and output.
+"""
+# Author: Gael Varoquaux, Alexandre Abraham, Philippe Gervais
+# License: simplified BSD
+
+import copy
+import gc
+import collections
+from six import string_types
+
+import numpy as np
+
+import nibabel
+
+from .numpy_conversions import as_ndarray
+
+
+def _get_shape(img):
+    # Use the fact that Nifti1Image has a shape attribute that is
+    # faster than loading the data from disk
+    if hasattr(img, 'shape'):
+        shape = img.shape
+    else:
+        shape = img.get_data().shape
+    return shape
+
+
+def _safe_get_data(img):
+    """ Get the data in the image without having a side effect on the
+        Nifti1Image object
```

Comment:
Should "Nifti1Image object" be replaced by "Image object" or "SpatialImage object"?